src/scene/graph-node.js

```diff
@@ -14,6 +14,13 @@ var scaleCompensateScale = new Vec3();
 var scaleCompensateScaleForParent = new Vec3();
 var tmpMat4 = new Mat4();
 var tmpQuat = new Quat();
+var position = new Vec3();
+var invParentWtm = new Mat4();
+var rotation = new Quat();
+var invParentRot = new Quat();
+var matrix = new Mat4();
+var target = new Vec3();
+var up = new Vec3();
```

Comment:
We can probably merge some of these globals into a minimal set of temp vec/quat/mats.